### DISCUSSION TITLE
I like this tool, thank you!

### DISCUSSION THREAD

**Discussion by DukeOwlington** [sonik-br/GDMENUCardManager#39] [Show and tell]
Hi guys, just wanted to stop by and say thank you for developing this tool. It is very convinient and simple. It makes uploading and managing Sega Dreamcast games on an SD Card very simple. Special thanks for maintaining the GNU/Linux version, because I avoid using propriatory OSes at all cost. Me and my family enjoy playing Dreamcast games and this tool made this little hobby of ours a bit easier. Thank you!

P.S. I run GDMENUCardManager on a Gentoo distro. I have to set DOTNET_SYSTEM_GLOBALIZATION_INVARIANT environment variable to 1 if I want it to launch like this: `export DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=1`, otherwise it complains on missing ICU dependency. Installing ICU doesn't help, but that's not a big deal for me. Otherwise GDMENUCardManager works very well.

![2023-04-13_10-51](https://user-images.githubusercontent.com/24269409/231647887-90cb254f-090d-4111-82d3-49f43204a7ed.png)
![signal-2023-04-13-092654_002](https://user-images.githubusercontent.com/24269409/231647920-7f081ac6-0bc9-4ecf-a6dc-8b2655b87b8f.jpeg)

**Comment by sonik-br** [comment]
I'm glad that it's working for you.
I do my best to keep the app simple and reliable.
Thank you for the comment!

I plan to update it to the latest version of dot net. Let's see if will make it more compatible and easy to use on all OSes.

**Comment by thamulot** [comment]
Yup, I'm stopping by to say thank you as well. Version 2 is awesome. I could finally make openMenu work which I find WAAAAAAYYY better than GDmenu 🎉

Thanks again!
